api/Dockerfile

```diff
@@ -1,13 +1,32 @@
-FROM python:3.12.8-alpine3.20 AS build
+FROM python:3.12.10-slim-bookworm AS build
 
 LABEL maintainer="https://github.com/prowler-cloud/api"
 
-# hadolint ignore=DL3018
-RUN apk --no-cache add gcc python3-dev musl-dev linux-headers curl-dev
+ARG POWERSHELL_VERSION=7.5.0
+
+# hadolint ignore=DL3008
+RUN apt-get update && apt-get install -y --no-install-recommends wget \
```

Comment:
Powershell needs `libicu72` to work. It was removed [here](https://github.com/prowler-cloud/prowler/pull/7563/commits/f369a66db0e6d874badc0ad690849ec6d01107ef).

Reply:
Yes, the Dockerfile part will change as we are looking to use the same procedure as in the CLI, we are waiting for this